src/helpers/configHelpers.ts

```diff
@@ -127,13 +127,19 @@ export function deleteConfigsFromUnkownServers(client: Client): void {
 		return;
 	}
 
-	const configFiles = fs.readdirSync(CONFIGS_PATH);
-	configFiles.forEach(file => {
-		const guildId = file.split(".")[0];
-		if (!client.guilds.cache.has(guildId)) {
-			resetConfigToDefault(guildId);
-		}
-	});
+	try {
+		const configFiles = fs.readdirSync(CONFIGS_PATH);
+		configFiles.forEach(file => {
+			const guildId = file.split(".")[0];
+			if (!client.guilds.cache.has(guildId)) {
+				resetConfigToDefault(guildId);
+			}
+		});
+	}
+	catch {
+		// Configs directory not present
+		return;
+	}
```

Comment:
Let's not use try/catch for control flow!
I hope the indentation is right, this was a pain to do on GitHub

```suggestion
	if (!fs.existsSync(CONFIGS_PATH)) return;

	const configFiles = fs.readdirSync(CONFIGS_PATH);
	configFiles.forEach(file => {
		const guildId = file.split(".")[0];
		if (!client.guilds.cache.has(guildId)) {
			resetConfigToDefault(guildId);
		}
	});
}
```

Reply:
That last curly looks ominous

Reply:
You're not guaranteed to have the guild in the cache here, are we?

Reply:
`.fetch` will take from cache first then make API request to fetch guild, preferred in scenarios like this. Failure to fetch == bad guild, remove.

not sure if this is a major concern here though?